Dockerfile

```diff
@@ -6,7 +6,8 @@ WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
-RUN CGO_ENABLED=0 GOOS=linux go build -o powd cmd/powd/main.go 
+RUN go build -modfile .bingo/govvv.mod -o=. github.com/ahmetb/govvv
+RUN pkg="$(go list ./buildinfo)" && govvvflags="$(./govvv -flags -pkg $pkg)" && CGO_ENABLED=0 GOOS=linux go build -ldflags="$govvvflags" -o powd cmd/powd/main.go
```

Comment:
build up the govvv flags to build with and build. this way docker builds get as much build info as possible embedded into them even though the person running the docker image may be building powergate from source.